x-pack/plugins/index_lifecycle_management/__jest__/client_integration/edit_policy/edit_policy.test.ts

```diff
@@ -127,7 +125,7 @@ describe('<EditPolicy />', () => {
         await actions.hot.setBestCompression(true);
         await actions.hot.toggleShrink(true);
         await actions.hot.setShrink('2');
-        await actions.hot.setReadonly(true);
+        await actions.hot.toggleReadonly(true);
```

Comment:
I've realised that these toggle/set functions do not actually set the boolean value, they just toggle whatever the toggle state is relative to what it was before. Not something to action, just observing since I introduced the boolean arg here.

Might just need some slight investigation to fix, @sebelga have you encountered this?

Reply:
Thanks for pointing this out @jloleysens! It's worth investigating for sure, I'll put it in an issue.